packages/ateam-cli/cmd/board-claim_claimItem.go

```
@@ -86,9 +90,11 @@ func init() {
 	boardClaimCmd.AddCommand(boardClaimClaimItemCmd)
 	boardClaimClaimItemCmd.Flags().StringVar(&boardClaimClaimItemCmdBody, "body", "", "Raw JSON body (overrides individual flags)")
 	boardClaimClaimItemCmd.Flags().StringVar(&boardClaimClaimItemCmdBodyFile, "body-file", "", "Path to JSON file to use as request body")
-	boardClaimClaimItemCmd.Flags().StringVar(&boardClaimClaimItemCmd_agent, "agent", "", "(Hannibal|Face|Murdock|B.A.|Amy|Lynch|Stockwell|Sosa|Tawnia)")
+	// Help text and completion mirror the validate.Enum list in RunE — no
+	// Sosa or Frankie (see the comment there before widening either).
+	boardClaimClaimItemCmd.Flags().StringVar(&boardClaimClaimItemCmd_agent, "agent", "", "(Hannibal|Face|Murdock|B.A.|Amy|Lynch|Stockwell|Tawnia)")
 	boardClaimClaimItemCmd.RegisterFlagCompletionFunc("agent", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

golangci-lint (2.12.2)

[error] Error return value of `boardClaimClaimItemCmd.RegisterFlagCompletionFunc` is not checked

(errcheck)